### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mariiasirenko2**

### CODE REVIEW

**Comment by knurff** [comment]
> There is no coverage information present for the Files changed

|Total Project Coverage|51.5%|:green_apple:|
|:-|:-:|:-:|

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# WE STARTED!
\ No newline at end of file
+# Interview Planning Application
+## NPE-Hunders team
```
Comment (ram333n) [inline README.md:2]: *Hunters

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+### Setting up the project
+
+Setup a project in docker:
+In order to create a project image run:
+
+`docker build -t "NAME:Dockerfile"`
+
+The project perform two dockerfiles:
+- `Dockerfile` - basic file to create an image 
+- `Dockerfile.db` - file to create an image of database based on PostgreSQL and filled with start sql datta
```
Comment (ram333n) [inline README.md:61]: *data

File: README.md [hunk 3]
```diff
@@ -1 +1,70 @@
...
+- `Dockerfile.db` - file to create an image of database based on PostgreSQL and filled with start sql datta 
+
+In order to run an application with docker compose, run the following:
+
+`docker-compose -f docker-compose.yml up`
+
+The project perform three docker compose files:
+- `docker-compose.db-only` - docker compose for postgres db with test data
+- `docker-compose.api-build` - same as db-only, but also build api and create connection between api and db
+- `docker-compose.api-pull` - same as api-build, but exept building api image, pulls image from dockerhub repo
```
Comment (ram333n) [inline README.md:70]: *except

File: README.md [hunk 4]
```diff
@@ -1 +1,70 @@
...
+
+#### Guest (no authenticated user)
+- `GET /weeks/current` - get current week number
+- `GET /weeks/next` - get next week number
+- `POST /auth/login` - authentification
+
+#### Interviewer (user with auth role: INTERVIEWER)
+- `POST /interviewers/<interviewerId>/slots` - create interviewer slot
+- `POST /interviewers/<interviewerId>/slots/{slotId}` - update interviewer slot
+- `GET /interviewers/<interviewerId>/slots` - get all interviewer slot
```
Comment (ram333n) [inline README.md:31]: *slots

File: README.md [hunk 5]
```diff
@@ -1 +1,70 @@
...
+- `POST /candidates/current/slots/{slotId}` - update candidate slot
+- `GET /candidates/current/slots` - get all candidate slots
+
+#### Coordinator (user with auth role: COORDINATOR)
+- `GET /weeks/{weekId}/dashboard` - get all interviewer and candidate slots
+- `POST /interviewers/{interviewerId}/slots/{slotId}` - update interviewer slot
+- `POST /bookings` - create booking
+- `POST /bookings/{bookingId}` - update booking
+- `DELETE /bookings/{bookingId}` - delete booking
+- `POST /users/interviewers` - grand Interviewer role
```
Comment (ram333n) [inline README.md:45]: *grant

File: README.md [hunk 6]
```diff
@@ -1 +1,70 @@
...
+- `GET /users/interviewers` - get list of interviewers 
+- `DELETE /users/interviewers/<interviewer-id>` - revoke interviewer role
+- `POST /users/coordinators` - grand Coordinator role
```
Comment (ram333n) [inline README.md:48]: *grant

File: README.md [hunk 7]
```diff
@@ -1 +1,70 @@
...
+- Create bookings for Interviewer and Candidate slots as Coordinator
+
+### API
+### All users
+- `GET /me` - return user info
+- `POST /auth/login` - authentication
+
+#### Guest (no authenticated user)
+- `GET /weeks/current` - get current week number
+- `GET /weeks/next` - get next week number
```
Comment (ram333n) [inline README.md:25]: It's better to move it to `All authorized users`, because Guest can invoke `POST /auth/login` only

File: README.md [hunk 8]
```diff
@@ -1 +1,70 @@
...
+
+The interview planning application is a RESTful service that performs communication and interview scheduling for interviewers and candidates via coordinators.
+
+The application supports the following functionality:
+- Create slots for Interviewer.
+- Create slots for  Candidate
+- Create bookings for Interviewer and Candidate slots as Coordinator
+
+### API
+### All users
```
Comment (ram333n) [inline README.md:19]: It's better to rename to `All authorized users`

File: README.md [hunk 9]
```diff
@@ -1 +1,70 @@
-# WE STARTED!
\ No newline at end of file
+# Interview Planning Application
+## NPE-Hunders team
+
+### Overview
+* [About the project](#about-the-project)
+* [API](#api)
+* [Setting up the project](#setting-up-the-project)
+
```
Comment (ram333n) [inline README.md:8]: It's better to add tech stack

**Comment by knurff** [comment]
> There is no coverage information present for the Files changed

|Total Project Coverage|51.4%|:green_apple:|
|:-|:-:|:-:|

File: README.md [hunk 10]
```diff
@@ -1 +1,77 @@
...
+The interview planning application is a RESTful service that provides communication and interview scheduling capabilities for interviewers and candidates via coordinators.
+
+The application supports the following functionality:
+- Create slots for Interviewer
+- Create slots for Candidate
+- Create bookings for Interviewer and Candidate slots as Coordinator
+
+Clarification on terms used:
+- Guest is an authenticated person
```
Comment (ram333n) [inline README.md:19]: On the contrary,` non-authenticated`

**Comment by knurff** [comment]
> There is no coverage information present for the Files changed

|Total Project Coverage|51.05%|:green_apple:|
|:-|:-:|:-:|

**Comment by knurff** [comment]
> There is no coverage information present for the Files changed

|Total Project Coverage|51.05%|:green_apple:|
|:-|:-:|:-:|

**Review by Famezzs — APPROVED** [review]
Good enough. Well done.

**Pull request merged**